department-of-veterans-affairs/va.gov-cms
# VACMS-8118 update wysiwyg styles to fix font regression

## Description

Closes #8118. Reverts the mixin updates I made when converting between themes for the wysiwyg styles only. There were some changes to button colors and fonts used in headings that shouldn't have been applied inside of wysiwygs. Mea culpa!

## Testing done
- Verified what wysiwyg looks like on prod & local vagovclaro
- Checked what wysiwyg looks like in local seven/vagovadmin
- Updated styles to match.

## Screenshots
Currently on prod:
<img width="1015" alt="Screen Shot 2022-02-23 at 9 57 12 AM" src="https://user-images.githubusercontent.com/11279744/155380434-bac3560a-6367-4cc2-9f46-4d18570c0d78.png">


Fixed:
<img width="937" alt="Screen Shot 2022-02-23 at 10 00 10 AM" src="https://user-images.githubusercontent.com/11279744/155380468-3efc83ca-a6c5-4460-821a-1b36a62f0a2c.png">


Old theme:
<img width="1217" alt="Screen Shot 2022-02-23 at 10 00 53 AM" src="https://user-images.githubusercontent.com/11279744/155380506-03682e56-5ffe-4ff9-ae83-7e0062a6ffcd.png">


## QA steps
as user with admin, visit `node/43580/edit` and confirm the VAMC service description field has correct styles in the wysiwyg (most noticeable on the header)


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
